src/qt/veil/settings/settingsfaq.cpp

```diff
@@ -11,6 +11,7 @@
 #include <qt/veil/settings/settingsfaq09.h>
```

Comment:
While I don't feel it's necessary to go into other files and add the missing copyrights (out of scope for this PR), I would like to see it added to this module since it was touched.